_data/mentors.yml

```diff
@@ -10,6 +10,11 @@
 #   lat: Mentor's Latitude
 #   lng: Mentor's Longitude
 
+- name: Dravit Lochan
+  github: DravitLochan
+  image: Dravit Lochan.jpg
+
+
```

Comment:
@Vishnu2875  Remove this extra line else LGTM 👍 .

Reply:
Done